code/modules/ninja/suit/n_suit_verbs/ninja_adrenaline.dm

```diff
@@ -1 +1,21 @@
-// TODO
\ No newline at end of file
+//Wakes the user so they are able to do their thing. Also injects a decent dose of radium.
+//Movement impairing would indicate drugs and the like.
+/obj/item/clothing/suit/space/space_ninja/proc/ninjaboost()
+
+	if(!ninjacost(0,N_ADRENALINE))
+		var/mob/living/carbon/human/H = affecting
+		H.SetParalysis(0)
+		H.SetStunned(0)
+		H.SetWeakened(0)
+		H.adjustStaminaLoss(-75)
+		H.stuttering = 0
+		H.say(pick("A CORNERED FOX IS MORE DANGEROUS THAN A JACKAL!","HURT ME MOOORRREEE!","IMPRESSIVE!"))
+		a_boost--
+		to_chat(H, "<span class='notice'>There are <B>[a_boost]</B> adrenaline boosts remaining.</span>")
+		s_coold = 3
+		addtimer(CALLBACK(src, .proc/ninjaboost_after), 70)
+
+/obj/item/clothing/suit/space/space_ninja/proc/ninjaboost_after()
+	var/mob/living/carbon/human/H = affecting
+	H.reagents.add_reagent("radium", a_transfer)
```

Comment:
Will have to balance adjust these numbers; TG's radium only irradiates at a rate of 2 instead of our 4; additionally, our radiation gives burn damage; may just make this a generic toxin.